eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/contig/ContigMapping.java

```diff
@@ -129,36 +98,24 @@ private void normalizeNames(ContigSynonyms contigSynonyms) {
         }
         if (contigSynonyms.getUcsc().equals(NOT_AVAILABLE)) {
             contigSynonyms.setUcsc(null);
-        } else {
-            contigSynonyms.setUcsc(removePrefix(contigSynonyms.getUcsc()));
-        }
-    }
-
-    private String removePrefix(String contig) {
-        Matcher matcher = PATTERN.matcher(contig);
-        String contigNoPrefix = contig;
-        if (matcher.matches()) {
-            contigNoPrefix = matcher.group(2);
         }
-        return contigNoPrefix;
     }
 
     public ContigSynonyms getContigSynonyms(String contig) {
```

Comment:
I assume RefSeq is prioritized because we will most often find those among list of contigs while importing?